QGL/PulsePrimitives.py

```diff
@@ -672,16 +673,17 @@ def CNOT_CR(controlQ, targetQ, **kwargs):
 
     if edge.isforward(controlQ, targetQ):
         # control and target for CNOT and CR match
-        return ZX90_CR(controlQ, targetQ, **kwargs) + \
+        seq = ZX90_CR(controlQ, targetQ, **kwargs).seq + \
```

Comment:
I don't see an easy alternative, unless we allow for CompoundGates of CompoundGates? Anyway, the occasional `.seq` doesn't bother me